frontend/src/components/SimulationNode/SimulationNode.tsx

```diff
@@ -27,6 +27,11 @@ export const SimulationNode: React.FC<SimulationNodeProps> = (props) => {
   });
 
   const updateNodePosition = (event: DraggableEvent, data: DraggableData) => {
+    //this condition prevent to trigger this function when user does not move the node,
+    // not trigger when doubleClick
+    //TODO not clever solution, search new one
+    if (positionX === data.x && positionY === data.y) return;
+    //Send new position of the node
```

Comment:
Çift tıklama sırasında ufacık bir pozisyon değşimi olması bile yine eski davranışa neden olur. Quire task'ına yazdığım gibi ya `grid` seçeneği ile çözülmesi gerekiyordu, ya da drag için minimum bir threshold belirleyip mesafe ölçümü yapman gerekliydi.

Ama bunu şimdilik kabul edelim, tekrar uğraşacak vaktimiz yok. Sonradan boşluk olursa geri dönülebilir.